agent/utils/dockerapi_compare_versions.go

```diff
@@ -0,0 +1,127 @@
+// Copyright 2014-2016 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//	http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package utils
+
+import   (
+	"fmt"
+	"strconv"
+	"strings"
+)
+
+type Docker_version string
```

Comment:
you should use mixed caps naming following the guideline [here](https://github.com/golang/go/wiki/CodeReviewComments#mixed-caps), same below